sentry-core/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "sentry-core"
+version = "0.1.0"
```

Comment:
I still think that `sentry-core` and `sentry-types` for that matter should get to 1.0, and stay there. So that integration authors can rely on that version.
Most users will rather use the `sentry` facade, which might have more breaking changes. But I have no strong objections to synchronize the versions either.